### PR TITLE
Move Notifier bridges doc to the code repo README files

### PR DESCRIPTION
**Pull request by fabpot**
Follow-up https://github.com/symfony/symfony/pull/49782
Now that the README files have the docs, let's remove them from here.

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -228,18 +228,18 @@ integration with these chat services:
 Service                                  Package                               DSN
 =======================================  ====================================  =============================================================================
 `AmazonSns`_                             ``symfony/amazon-sns-notifier``       ``sns://ACCESS_KEY:SECRET_KEY@default?region=REGION``
-:doc:`Discord <notifier/discord>`        ``symfony/discord-notifier``          ``discord://TOKEN@default?webhook_id=ID``
+`Discord`                                ``symfony/discord-notifier``          ``discord://TOKEN@default?webhook_id=ID``
```
Comment (alamirault) [inline notifier.rst]:
```suggestion
`Discord`_                               ``symfony/discord-notifier``          ``discord://TOKEN@default?webhook_id=ID``
```

File: notifier.rst [hunk 2]
```diff
@@ -228,18 +228,18 @@ integration with these chat services:
...
 `FakeChat`_                              ``symfony/fake-chat-notifier``        ``fakechat+email://default?to=TO&from=FROM`` or ``fakechat+logger://default``
 `Firebase`_                              ``symfony/firebase-notifier``         ``firebase://USERNAME:PASSWORD@default``
 `Gitter`_                                ``symfony/gitter-notifier``           ``gitter://TOKEN@default?room_id=ROOM_ID``
 `GoogleChat`_                            ``symfony/google-chat-notifier``      ``googlechat://ACCESS_KEY:ACCESS_TOKEN@default/SPACE?thread_key=THREAD_KEY``
 `LinkedIn`_                              ``symfony/linked-in-notifier``        ``linkedin://TOKEN:USER_ID@default``
 `Mattermost`_                            ``symfony/mattermost-notifier``       ``mattermost://ACCESS_TOKEN@HOST/PATH?channel=CHANNEL``
 `Mercure`_                               ``symfony/mercure-notifier``          ``mercure://HUB_ID?topic=TOPIC``
-:doc:`MicrosoftTeams <notifier/teams>`   ``symfony/microsoft-teams-notifier``  ``microsoftteams://default/PATH``
+`MicrosoftTeams`                         ``symfony/microsoft-teams-notifier``  ``microsoftteams://default/PATH``
```
Comment (alamirault) [inline notifier.rst]:
```suggestion
`MicrosoftTeams`_                        ``symfony/microsoft-teams-notifier``  ``microsoftteams://default/PATH``
```

File: notifier.rst [hunk 3]
```diff
@@ -228,18 +228,18 @@ integration with these chat services:
...
 `RocketChat`_                            ``symfony/rocket-chat-notifier``      ``rocketchat://TOKEN@ENDPOINT?channel=CHANNEL``
-:doc:`Slack <notifier/slack>`            ``symfony/slack-notifier``            ``slack://TOKEN@default?channel=CHANNEL``
-:doc:`Telegram <notifier/telegram>`      ``symfony/telegram-notifier``         ``telegram://TOKEN@default?channel=CHAT_ID``
+`Slack`                                  ``symfony/slack-notifier``            ``slack://TOKEN@default?channel=CHANNEL``
```
Comment (alamirault) [inline notifier.rst]:
```suggestion
`Slack`_                                 ``symfony/slack-notifier``            ``slack://TOKEN@default?channel=CHANNEL``
```

File: notifier.rst [hunk 4]
```diff
@@ -228,18 +228,18 @@ integration with these chat services:
...
+`Telegram`                               ``symfony/telegram-notifier``         ``telegram://TOKEN@default?channel=CHAT_ID``
```
Comment (alamirault) [inline notifier.rst]:
```suggestion
`Telegram`_                              ``symfony/telegram-notifier``         ``telegram://TOKEN@default?channel=CHAT_ID``
```

**Review by alamirault — CHANGES_REQUESTED** [review]
Missing `_` for links

File: notifier.rst (hunk 1 above)
Comment (fabpot) [reply notifier.rst]: oops, fixed them all now.

**Review by alamirault — APPROVED** [review]

**Pull request merged**